src/richie/apps/courses/templates/courses/cms/subject_detail.html

```diff
@@ -4,49 +4,50 @@
 {% block content %}
 <div class="subject-detail">
```

Comment:
You may add templatetag `{% spaceless%}...{% endspaceless %}` around content just under this div, so HTML is a bit lighter and unnecessary whitespaces won't never make tests fails.

Reply:
I ended up not adding the spaceless tag after all.

Reply:
and because ... ?

Reply:
I was trying to stay consistent with other templates.
I added them now.
Thanks for the review.